core/src/main/java/com/novoda/noplayer/model/AudioTracks.java

```diff
@@ -0,0 +1,78 @@
+package com.novoda.noplayer.model;
+
+import com.novoda.noplayer.internal.exoplayer.mediasource.AudioTrackType;
+import com.novoda.utils.Optional;
+
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.List;
+
+public class AudioTracks implements Iterable<PlayerAudioTrack> {
+
+    private final List<PlayerAudioTrack> audioTracks;
+
+    public static AudioTracks from(List<PlayerAudioTrack> audioTracks) {
+        return new AudioTracks(Collections.unmodifiableList(audioTracks));
+    }
+
+    private AudioTracks(List<PlayerAudioTrack> audioTracks) {
+        this.audioTracks = audioTracks;
+    }
+
+    public boolean containsPlayerAudioTrackWith(AudioTrackType trackType) {
```

Comment:
I don't create many first class collections and was unsure whether putting these here was a good idea. I would be interested in hearing other peoples thoughts on this. 

As a side note, these methods are fully tested 😄